src/thunderbots/shared_util/constants.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <string>
+#include "../software/ai/world/team.h"
+
+// Constants for ROS nodes, message, and topics
+static const std::string BACKEND_INPUT_BALL_TOPIC          = "backend/ball";
+static const std::string BACKEND_INPUT_FIELD_TOPIC         = "backend/field";
+static const std::string BACKEND_INPUT_FRIENDLY_TEAM_TOPIC = "backend/friendly_team";
+static const std::string BACKEND_INPUT_ENEMY_TEAM_TOPIC    = "backend/enemy_team";
+static const std::string AI_PRIMITIVES_TOPIC               = "backend/enemy_team";
```

Comment:
Does firmware need to know about these constants? If not we should put them in the software folder. Either way, I don't think we'll be able to build in `std::string` to firmware, so these will probably have to change.